README.md

```diff
@@ -432,12 +473,23 @@ Sine LFO.
 
 Sine oscillator.
 
-- Inputs: `FM`
+- Inputs: `FM`, `LinFM`, `PM`
 - Outputs: `Out`
 - Parameters:
 	- `Range`
 	- `Tune`
 	- `FM`
+	- `LinFM`
+	- `PM`
+	
+### Slew
+
+Slew rate limiter.
+
+- Inputs: `In`
+- Outputs: `Out`
+- Parameters:
+	- `Time` (seconds)
```

Comment:
would you mind changing unit to milliseconds? Reason: I use milliseconds for time for other modules.

Reply:
sure!